docs/computing/containers/creating.md

```diff
@@ -76,3 +76,54 @@ Below is a table of common docker base images and whether installing simple pack
 |ubuntu|16.04-22.04|yes|
 
 Some issues related to, for example, glibc, fakeroot, file permissions and old remote repos are often difficult to solve, so trying out a few different base images can be a good idea before spending a lot of time debugging.
+
+## Using GPU from containers in interactive sessions in Puhti
+
+### Running existing images
+
+To run programs that [use GPU](https://apptainer.org/docs/user/latest/gpu.html) use `--nv` flag when starting the container `apptainer run --nv /path_to_image/image.sif`. To use the graphical display with [VirtualGL](https://virtualgl.org/) a few environment variables have to be set as well. In the base images provided these are set automatically when the container is started. (Note that if you run `apptainer shell` instead of `apptainer run` `%runscript` is not executed and necessary environment variables for vgl are not set, you then have to set them manually, see [base image definition files](https://github.com/CSCfi/singularity-recipes/visualization) for details.)
```

Comment:
"base images provided" is somewhat unclear. I.e. what are they, who are they provided by, and where can they be found (explained in the next section)?
Might be good to have a separate section/paragraph just explaining e.g. what the base images are and why to use them.